### DISCUSSION TITLE
Difference between the Maximum Capacity and Macos Capacity

### DISCUSSION THREAD

**Discussion by Yolo9958** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1242] [Q&A]
I want to know the difference between Maximum Capacity and Macos Capacity. In the system info we have battery health at 95%, but in some applications it's 91.
![CleanShot 2024-06-17 at 16 25 25@2x](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/57700443/08506287-c6fb-4ca2-b5b7-b520aa0066e7)

**Comment by MatthiasKerbl** [comment]
Hi @Yolo9958,

The Battery Management System reports the Maximum Capacity, which fluctuates up and down depending on how well your battery is calibrated. MacOS reports the macOS Capacity and we do not have any insights into how Apple came up with the number. However, it tends to be much more stable than the Maximum Capacity.

Best,
Matthias
